staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -330,6 +330,10 @@ type ClaimMappings struct {
 	// The claim's value must be a singular string.
 	// Same as the --oidc-username-claim and --oidc-username-prefix flags.
 	// If username.expression is set, the expression must produce a string value.
+	// If username.expression uses 'claims.email', then 'claims.email_verified' must be used in
+	// username.expression or extra[*].valueExpression or claimValidationRules[*].expression.
```

Comment:
Could we add [integrations tests](https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/oidc/oidc_test.go#L561) with using `claims.email_verified` in
1. username.expression
2. extra[*].valueExpression
3. claimValidationRules[*].expression

Reply:
I can though we can only cover the positive cases since the API server won't start on the negative cases.

Reply:
I don't feel strongly about unit or integration tests as long as the unit tests actually exercise the functional evaluator

Reply:
Opened #123825, will handle this in a follow-up due to limited time left in the release.